backend/server/package.json

```diff
@@ -21,6 +21,7 @@
   "dependencies": {
     "bcryptjs": "^2.4.3",
     "cookie-parser": "^1.4.6",
+    "cors": "^2.8.5",
```

Comment:
This needs types as well!

https://www.npmjs.com/package/@types/cors

FYI: If you are installing a package from the npm registry, you can see if the types are required by seeing the DT (Definitely Typed) icon at the top.

![image](https://github.com/Koodikukkaro/Kendo-tournament-app/assets/82541244/b6e7c434-e9bc-4184-a7d0-5a1d29b302ba)

Reply:
Cors typed added.